connectors/v2/ego-fm.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#playerInfo';
+
+Connector.artistSelector = '#current > span.artist';
+
+Connector.trackSelector = '#current > span.song';
+
+Connector.isPlaying = function () {
```

Comment:
You could also use the `Connector.playButtonSelector` setting this to `#play`. Have a look at: https://github.com/david-sabata/web-scrobbler/blob/master/core/content/connector.js#L77